torch/onnx/symbolic_opset9.py

```diff
@@ -846,6 +847,9 @@ def _standard_gamma(g: jit_utils.GraphContext, self, generator):
 @_onnx_symbolic("aten::t")
 @_beartype.beartype
 def t(g: jit_utils.GraphContext, self):
+    rank = symbolic_helper._get_tensor_rank(self)
+    if rank is None or rank < 2:
+        return g.op("Identity", self)
```

Comment:
According to [numpy.transpose](https://numpy.org/doc/stable/reference/generated/numpy.transpose.html), they claim `Transposing a 1-D array returns an unchanged view of the original array.` I think we should raise an issue there if this corner case is not handled by itself?

Reply:
Are you suggesting Identity is not needed? Can you elaborate?

Reply:
I was saying that if the intention of this op was to cover numpy.transpose, we should raise an issue to ONNX also. But here, I don't know we should take noop or identity. I think either way users are not getting what they wanted.

Reply:
Ah that makes sense. Having onnx to support it sounds like a good idea.

Reply:
The current version should work. I will add a comment

Reply:
Done